src/examples/notifications/global-alert/GlobalAlertTypes.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { GlobalAlert } from '@zendeskgarden/react-notifications';
+import { Anchor } from '@zendeskgarden/react-buttons';
+
+const StyledSpacer = styled.div`
+  height: ${p => p.theme.space.md};
+`;
+
+const Example = () => (
+  <>
+    <GlobalAlert type="info">
+      <GlobalAlert.Title>New update available</GlobalAlert.Title>
+      <GlobalAlert.Content>
+        <span>Your account will automatically update in 5 days. </span>
+        <Anchor href="#" isExternal>
+          Find out more
+        </Anchor>
```

Comment:
```suggestion
        Your account will automatically update in 5 days.{' '}
        <Anchor href="#" isExternal>
          Find out more
        </Anchor>
```

Prefer simplified DOM structure.